src/main/java/com/minecolonies/blocks/AbstractBlockHut.java

```diff
@@ -174,11 +188,35 @@ public void onBlockPlacedBy(@NotNull World worldIn, @NotNull BlockPos pos, IBloc
         }
     }
 
+    @Override
+    public boolean isFullCube(final IBlockState state)
+    {
+        return false;
+    }
+
+    @Override
+    public boolean isOpaqueCube(final IBlockState state)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)